projects/storefrontapp-e2e-cypress/cypress/integration/regression/anonymous-consents/anonymous-consents-config-flow.e2e.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  anonoymousConsentConfig,
+  anonymousConfigTestFlow,
+  MARKETING_NEWSLETTER,
+  PERSONALIZATION,
+  sessionLogin,
+  showAnonymousConfigTest,
+  STORE_USER_INFORMATION,
+} from '../../../helpers/anonymous-consents';
+
+context('Anonymous consents - config flow', () => {
+  describe('when config hides legalDescription is false and showAnonymousConsents is false', () => {
+    before(() => {
+      cy.window().then(win => {
+        win.sessionStorage.clear();
+        win.localStorage.clear();
+      });
+
+      anonoymousConsentConfig(MARKETING_NEWSLETTER, false, [], {
```

Comment:
To improve readability of the code, maybe we can define a constant `const showLegalDescriptionInDialog=false` and then pass that to the function.

Reply:
I like that you used different template IDs in this and in the test below 👍